db.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
db.json was seperated into a server, so this is all that's still necessary in the old file.